mkroosevelt.js

```diff
@@ -45,7 +51,14 @@ else if (cmd === '-v' || cmd === '--v' || cmd === '-version' || cmd === '--versi
 }
 else if (cmd) {
   try {
-    createSampleApp();
+    if (cmd === '.') {
+      currentDirectory = true;
+      createSampleApp(currentDirectory);
```

Comment:
Yep - users should know any files with the same name will be overwritten. Ideally it should be smart and detect if this will happen and warn only if it will.